internal/db/schema/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql

```diff
@@ -2,97 +2,176 @@
 -- SPDX-License-Identifier: BUSL-1.1
 
 begin;
-  create table target_deleted (
+  create table target_tcp_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
+  create table target_ssh_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  -- Target repository list logic is implemented
+  -- against views, so use a view for the deleted
+  -- ids too.
+  -- Note: does not end in _deleted so as
+  -- not to be attached a trigger in the below function.
+  create view target_all_subtypes_deleted_view
+  as
+    select * from target_tcp_deleted
+    union
+    select * from target_ssh_deleted;
+
+  -- Sessions
   create table session_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table auth_method_deleted (
+  -- Auth methods
+  create table auth_oidc_method_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  create table auth_ldap_method_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  create table auth_password_method_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
+  -- Auth tokens
   create table auth_token_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table credential_library_deleted (
+  -- Auth accounts
+  create table auth_oidc_account_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table credential_deleted (
+  create table auth_ldap_account_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table credential_store_deleted (
+  create table auth_password_account_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table iam_group_deleted (
+  -- Auth managed groups
+  create table auth_oidc_managed_group_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  create table auth_ldap_managed_group_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  -- Credential libraries
+  create table credential_vault_library_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  create table credential_vault_ssh_cert_library_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table host_catalog_deleted (
+  -- Credentials
+  create table credential_static_json_credential_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
-  create table host_set_deleted (
+  create table credential_static_username_password_credential_deleted (
```

Comment:
I get the following 2 warnings:
```
psql:/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql:207: NOTICE:  identifier "credential_static_username_password_credential_deleted_delete_time_idx" will be truncated to "credential_static_username_password_credential_deleted_delete_t"
psql:/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql:207: NOTICE:  identifier "credential_static_ssh_private_key_credential_deleted_delete_time_idx" will be truncated to "credential_static_ssh_private_key_credential_deleted_delete_tim"
```

Can we use index names for these two tables that are under the postgresql naming limit?

Reply:
I'll truncate this table name so we can keep using the same loop below for the index and trigger creation, thanks

Reply:
Ah haha that's not going to work 😂. OK I'll think of something here.

Reply:
I added alter index statements at the end, but that doesn't stop the NOTICEs from appearing. If you would prefer we don't have the notices appear, I'll have to change these tables to not be part of the loop, which I can do of course. Let me know what you prefer 😄.

Reply:
This is fine